zebra-state/Cargo.toml

```diff
@@ -30,6 +30,7 @@ proptest-impl = [
 elasticsearch = [
     "dep:elasticsearch",
     "dep:serde_json",
+    "zebra-chain/elasticsearch",
```

Comment:
Optional: we could make the same change in `zebrad`

The feature resolver should work it out automatically, but it's had bugs in the past.

https://github.com/ZcashFoundation/zebra/blob/2e379811c3a8f4dd74c8e0e735738dcbe52d4288/zebrad/Cargo.toml#L37-L39